apps/duellen/app/components/Start.jsx

```diff
@@ -58,32 +71,60 @@ export default class Start extends React.Component {
     this.props.history.push('/Intro');
   }
 
+  // Loads more quizzes
+  loadMore(e){
+    e.preventDefault();
+    try {
+      // Gets the more quizzes from the backend        
+      fetch(backendURL + 'get/all/quizzes/as/json/?lowerlim=' + this.state.lowerLim.toString() + '&upperlim='+ this.state.upperLim.toString()
+          ).then(
```

Comment:
Ah, sorry for missing this when you asked me :D But I guess usually the indentation/new lines are done like this
```
fetch(...)
.then(...)
.then(...)
```
But I'm fine with this.

Reply:
We should really use a linter...